src/main.ts

```diff
@@ -24,14 +26,24 @@ const run = async (): Promise<void> => {
 
     const testResult = await processTestResults(resultsPath, allowFailedTests);
     comment += formatResultMarkdown(testResult);
-    summary += formatResultHtml(testResult, showFailedTestsOnly, showTestOutput);
+    summary += formatResultHtml(testResult, showFailedSuitesOnly, showFailedTestsOnly, showTestOutput);
 
     if (coveragePath) {
-      const testCoverage = await processTestCoverage(coveragePath, coverageType, coverageThreshold);
+      const testCoverage = await processTestCoverage(coveragePath, coverageType, coverageThreshold, changedFilesAndLineNumbers);
       comment += testCoverage ? formatCoverageMarkdown(testCoverage, coverageThreshold) : '';
       summary += testCoverage ? formatCoverageHtml(testCoverage) : '';
+      if (testCoverage) {
+        for(const myMod of testCoverage.modules) {
+          const changedFiles = myMod.files.filter(f => f.changedLinesTotal > 0);
+          if (changedFiles.length > 0) {
+            const tempComment = formatChangedFileCoverageMarkdown(changedFiles);
+            await publishComment(token, `${myMod.name}'s Changed File Coverage`, tempComment, postNewComment);
```

Comment:
so the changed file coverage should be a separate comment for each module? have you considered a single comment for all the changes?